imap_processing/tests/mag/test_mag_l1b.py

```diff
@@ -132,3 +136,33 @@ def test_mag_compression_scale(mag_l1a_dataset):
     # width of 14 should be multiplied by 4
     scaled_vectors = calibrated_vectors * 4
     assert np.allclose(output["vectors"].data[3][:3], scaled_vectors)
+
+
+def test_calibrate_vector():
+    # from MFOTOURFO
```

Comment:
What does this mean? Just curious. Can you add comment for future reference?

Reply:
Oh, that's the name of the variable in the calibration CDF file

Reply:
It stands for Measurement Frame Outboard to Universal Reference Frame Outboard. It represents the conversion matrix from the 2 reference frames.

Reply:
It would have been helpful to add that. A year from now, this clarification will be helpful